web/src/components/organisms/Asset/AssetList/hooks.ts

```diff
@@ -94,18 +97,16 @@ export default (projectId?: string) => {
               variables: { projectId, createdById: currentUser?.id, file },
             });
             if (result.errors || !result.data?.createAsset) {
-              // TODO: notification
-              // console.log("Failed to add one or more assets.");
+              Notification.error({ message: t("Failed to add one or more assets.") });
             }
           }),
         );
         if (results) {
-          // TODO: notification
-          // console.log("Successfully added one or more assets.");
+          Notification.success({ message: t("Successfully added one or more assets.") });
```

Comment:
"Successfully added one or more assets!"

Please check the other notification's language to make sure they are all in the same format/style!

Reply:
Done!

Reply:
` Failed xxx xxx.` use periods, not exclamation marks (!)

Reply:
Done!